### ISSUE TITLE
Ys I & II Chronicles issue: videos end too early on PPSSPP1.3 or more [MPEG] (due to being too fast)

### ISSUE THREAD

**Issue by LightVR0** [hrydgard/ppsspp#9454]
I try playing Ys I & II Chronicles but the intro videos are too fast (60/30). The sounds are a normal speed during the videos' shown. And not by adjust the Framerate or activate auto-frameskip. I tested this on Android 4.4.2 and Windows 7's laptop, both platforms with both stable versions 1.3.0.1 and 1.3.0, respectively.

**Comment by LightVR0** [comment]
No cheats/patches, only clean copies.

**Comment by unknownbrackets** [comment]
It seems to match the timing of this in all backends, including Direct3D9:

https://youtu.be/UrYcrK60QK0?t=2m57s

When you say you detect - what does PPSSPP say the FPS is?

-[Unknown]

**Comment by LightVR0** [comment]
This video showns at 30/30 (3:26). In my case, is too shorter (40/60 FPS and 102%) and at start point instead.

**Comment by unknownbrackets** [comment]
Maybe it's a setting issue - could you have the CPU Mhz overidden or something?

I let the entire video play and the audio was in sync the entire time for me.  To be clear, for you is the audio ending early (silence for part of the video), or is the audio ending late (the wrong sounds play near the end of the video)?

Have you tried with default settings?

-[Unknown]

**Comment by LightVR0** [comment]
On default settings (android 4.4.2) slowdown. I'm not tested on Windows due to bad connection.

**Comment by unknownbrackets** [comment]
"Timing hack" breaks things, it's a hack that breaks timing to try to make games run faster.

For me, on Android, with or without frameskip, and without any hacks, the audio plays perfectly in sync with the video.

If this only happens with the timer hack enabled, then it's not a bug.  That's timer hack doing it's job: breaking the game to make it run faster.  If it was just faster without breaking games, we'd make it checked by default and wouldn't make it possible to uncheck.

-[Unknown]

**Comment by LightVR0** [comment]
This bug ocurrs and the Timming hack is disabled. I tested on Windows 7 (laptop Olidata OX (J10IL1), Intel Atom 1.6 GHz x2, 1 GB RAM, 256 MB VRAM, Direct3D 9 (due to the internal graphics card, don't run Direct3D 11 or OpenGL 2+)) and Android 4.4.2 (Lanix Ilium X110)

**Comment by LightVR0** [comment]
same de-sync on 1.4.2

**Comment by LightVR0** [comment]
Same case on v1.5.2, too on Android 6

**Comment by LightVR0** [comment]
Same case on PPSSPP 1.5.4. Fix it, please!!!

**Comment by unknownbrackets** [comment]
Assuming you're using default settings, a clean disc, no hacks or cheats, and the same latest version - then there must be something causing it to behave differently.  Maybe savedata?

I'm not sure if anyone else has reproduced it, but I haven't.  I'm not even sure where to look, and since it doesn't happen for me, I can't even scare up any leads.

-[Unknown]

**Comment by LightVR0** [comment]
I think I also tried changing the CPU speed but there was no difference.

**Comment by Gamemulatorer** [comment]
How about this now? @LightVR0 no feedback outdated?

**Comment by BourgeoisBear** [comment]
I have experienced the same issue with this game running v1.14.4 on Linux.

**Comment by nassau-tk** [comment]
This issue still happen on v1.19.3-739 Android.

**Comment by insea190** [comment]
Even in the Windows version v1.20.2, movies are still playing at double speed.

**Comment by Freakservo** [comment]
Persists in 1.20.4.

**Comment by hrydgard** [comment]
Please re-test with v1.20.4-1845 or later (#22309).

**Comment by nassau-tk** [comment]
@hrydgard 

Oops!!
Please play the URL.

https://drive.google.com/file/d/1Lzptdu10DoJUSjqWcKR6vgOfooKmimH2/view?usp=sharing

The opening movie has become riddled with jagged glitches.

Test & Rec on SD8 gen2 + 1.20.4-1845.

And Windows + 1.20.4-1841 too.

remarks : 1845&1841 is same finish timing.

**Comment by mousep** [comment]
Seems the timing is correct now, but there are decode issues.

**Comment by mousep** [comment]
Seems not completely fixed yet. The video still plays a bit faster. The video of the YS1 OP movie will end several seconds before the music. And the frame pace seems to be unstable, where it will play smoothly for a while, then appears to have some stutter for several seconds, and then become smooth again. The smooth -> stutter -> smooth -> stutter -> ... loop will last during the playback.

**Comment by hrydgard** [comment]
@mousep that was tried with 1878 or later?

**Comment by mousep** [comment]
@hrydgard yes, tried with the latest development build ppsspp_win_v1.20.4-1887-gaeadfb31342b.zip

EDIT: The fix does improve the playback a bit. Before the video plays in double speed, while with fix it can slowdown to almost keep with them music. It seems during the "smooth" part, it doesn't keep the pace and then tries to slowdown during the "stutter" part.

**Comment by hrydgard** [comment]
ok. well, an improvement is an improvement, but I'll open this again. There's clearly something really weird going on in this game.
